neurom/core/tests/test_neuron.py

```diff
@@ -82,10 +85,6 @@ def check_cloned_neuron(nrn1, nrn2):
     nrn2.soma.radius = 10.
     nt.ok_(nrn1.soma.radius != nrn2.soma.radius)
 
-    nrn2._data.data_block[0, :] = np.zeros_like(nrn2._data.data_block[0, :])
```

Comment:
we don't need to test this anymore?

Reply:
data_block is an internal object and will no longer exist.
I've never been a fan of testing internal object because unit test should test the functionality and not the internality. Normally of course, I would not have deleted a unit test but this one is *kinda* useless and will be gone soon anyway.